frontend/src/components/widgets/Radio/Radio.tsx

```diff
@@ -80,7 +83,17 @@ class Radio extends React.PureComponent<Props, State> {
 
     return (
       <div className="row-widget stRadio" style={style}>
-        <StyledWidgetLabel>{this.props.element.label}</StyledWidgetLabel>
+        <StyledWidgetLabel style={{ justifyContent: "flex-start" }}>
```

Comment:
We should be able to move this into the styled component?

Reply:
It's just that `StyledWidgetLabel` is shared by other components, and we only want the `flex-start` here.

Is it possible to/should I create a styled component based on another styled component?

Reply:
I know you can style a component so I assumed you can style a styled component? @kmcgrady would know best but [here's how to style a component](https://emotion.sh/docs/styled#styling-any-component)